.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: go
 
 go:
-  - "1.9.x"
```

Comment:
Did we drop support for `1.9.x`?

Reply:
Yes. We dropped support for 1.10.x as well (in the dev branch). But this is enough for this branch to satisfy the CI requirements.